tools/functional-tester/etcd-runner/main.go

```diff
@@ -227,7 +227,7 @@ func runWatcher(eps []string) {
 	keyPrePrefix := 30              // max number of keyPrePrefixs for put operation
 
 	prefixes := generateUniqueKeys(5, noOfPrefixes)
-	keys := generateRandomKeys(10, keyPrePrefix)
+	keys := generateUniqueKeys(10, keyPrePrefix)
```

Comment:
@xiang90 sorry i misread it. I didn't actually understand what exactly you mean? What i had meant earlier was that even under current circumstances, random keys will work now, since we are doing revision check for not performingput twice for the same keys. So even now if random keys have duplicate keys it will work.